Automattic/wp-calypso
# Remvoing the sendinblue card from marketing tools

Closes  #78877

## Proposed Changes

* Removing the Sendinblue card from Marketing Tools. This product is rebranded, and the link 404s. See the above issue for context.

## Testing Instructions
* Via the Calypso.live link below, head over to: `/marketing/tools/:site_slug`
* You should not see the Sendinblue card anymore.
* Verify no console or network errors.

## Pre-merge Checklist
